src/ConditionalLogic.php

```diff
@@ -20,18 +20,19 @@ class ConditionalLogic
     public array $rules = [];
 
     public function __construct(
-        string|array $field,
+        string $name,
         string $operator,
-        mixed $value = null
+        mixed $value = null,
+        string|null $group = null
```

Comment:
Do we need to allow null if we default to null? I guess we should only allow string if the argument is present.

Reply:
Yes it’s a declaration error 👍

Reply:
Oh, I see. Then we'll leave it as is.

Reply:
Fixed in last commit